src/Link/Link.features.stories.tsx

```diff
@@ -19,3 +19,77 @@ export const Underline = () => (
     Link
   </Link>
 )
+
+export const Inline = () => (
```

Comment:
This seems like a good candidate for a dev story, similar to what we have for button (I think?). What would you think of having the example in the feature here be a link in a text block and this would be its own dev story?

Reply:
Good idea, done!